### PR TITLE
[14.0][MIG] website_sale_show_company_data: Migration to 14.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT32119

### CODE REVIEW

**Comment by ernestotejeda** [comment]
@chienandalu could you review this, please?

File: website_sale_show_company_data/__manifest__.py [hunk 1]
```diff
@@ -3,10 +3,10 @@
 {
     "name": "Website Sale Show Company Data",
     "summary": "Show commercial partner data if any",
-    "version": "12.0.1.1.0",
+    "version": "14.0.1.1.0",
```
Comment (pedrobaeza) [inline website_sale_show_company_data/__manifest__.py]:
```suggestion
    "version": "14.0.1.0.0",
```

**Review by pedrobaeza — APPROVED** [review]

File: website_sale_show_company_data/__manifest__.py (hunk 1 above)
Comment (ernestotejeda) [reply website_sale_show_company_data/__manifest__.py]: done

**Review by chienandalu — APPROVED** [review]
:+1:

**Pull request merged**